ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -95,6 +109,18 @@ class MySqlNamedBlobDb implements NamedBlobDb {
   private static final String LIST_QUERY = String.format("SELECT %1$s, %2$s, %3$s, %4$s FROM %5$s "
           + "WHERE (%6$s, %7$s) = (?, ?) AND %1$s LIKE ? AND %1$s >= ? ORDER BY %1$s ASC LIMIT ?", BLOB_NAME, BLOB_ID,
       EXPIRES_TS, DELETED_TS, NAMED_BLOBS, ACCOUNT_ID, CONTAINER_ID);
+  // @formatter:off
+  private static final String LIST_QUERY_V2 = String.format(""
```

Comment:
Can you add some comments here to explain what's your query is doing?

Reply:
Sure, Bei Xu is reviewing this query, if she is good with it, I can add a comment then.

Reply:
Bei reviewed this query and give it a go. Just added blob_name per Bei's suggestion, and added the java doc for this query.